src/pilot/dell_nfv.py

```diff
@@ -252,8 +258,14 @@ def edit_environment_files(
                     '\\" |" ' +
                     dpdk_file)
                 cmds.append(
-                    'sed -i "s|# IsolCpusList:.*|IsolCpusList: ' +
-                    self.nfv_params.isol_cpus + '|" ' + dpdk_file)
+                    'sed -i "s|IsolCpusList:.*|IsolCpusList: \\"' +
+                    self.nfv_params.isol_cpus + '\\" |" ' + dpdk_file)
+
+            # Change neuton-network-type to vlan. In future if we 
```

Comment:
neuton-network-type should be neutron-network-type

Reply:
Changed